Makefile.common

```diff
@@ -203,9 +203,13 @@ endif
 	$(QUIET_AS)$(AS) $(ASFLAGS) $(<) -o $@
 
 # Build rule for Windows resource files
-# TODO: Support dependency tracking
+dists/scummvm.rc: config.mk
+
 %.o: %.rc
 	$(QUIET)$(MKDIR) $(*D)
+	$(QUIET)$(MKDIR) $(*D)/$(DEPDIR)
+	$(QUIET)echo -n "$@: $< " > $(*D)/$(DEPDIR)/scummvm.d
```

Comment:
Given the complexity of this line and the following sed magic, I would suggest a comment to explain what this is intended to do.

Reply:
i.e. I have written a fair amount of sed scripts, but that level is a bit XKCD 208 :)

Reply:
😆